### PR TITLE
New accept and submit flow

### PR DESCRIPTION
**Pull request by slawosz**
Jira ticket: https://regulatorydelivery.atlassian.net/browse/COSBETA-1388
Review app: https://cosmetics-pr-2469-submit-web.london.cloudapps.digital/

New flow for accepting notification. Adds 3 new routes for `DraftController`:
* `review` - displays notification which is ready to submit. May display nanomaterials errors if there is nanomaterial not assigned to any item. Based on `show/edit` pages
* `declaration` - declaration form - new page 
* `accept` - moved from `NotificationsController#confirm`

Adds separate declaration  page:
Screenshot 2022-04-20 105220

### CODE REVIEW

File: cosmetics-web/app/analyzers/anti_virus_analyzer.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ def self.accept?(_blob)
   end
 
   def metadata
+    return { safe: true } if Rails.env.development?
+
```
Comment (scruti) [inline cosmetics-web/app/analyzers/anti_virus_analyzer.rb:8]: :scissors: This does not seem to belong to this PR?

Comment (scruti) [inline cosmetics-web/app/analyzers/anti_virus_analyzer.rb:8]: :scissors: This does not seem to belong to this PR?

**Review by scruti — APPROVED** [review]

**Review by nasirkhanbeis — APPROVED** [review]
Tested this on the review app, happy with the new accept and submit flow. Raised a couple of tickets to discuss as follow up to this pr. Thanks!
